textpipe/doc.py

```diff
@@ -274,3 +275,20 @@ def _set_default_nlp(self, lang):
             self._spacy_nlps[lang] = {}
         model = spacy.load('{}_core_{}_sm'.format(lang, 'web' if lang == 'en' else 'news'))
         self._spacy_nlps[lang][None] = model
+
+    @functools.lru_cache()
+    def sentiment(self, lang):
+        """
+        Returns sentiment score (-1 to 1) and a confidence level (0 to 1)
+
+        Currently only Dutch supported
+
+        >>> doc = Doc('Een leuke test zin.')
+        >>> doc.sentiment('nl')
+        (0.6, 0.9666666666666667)
+        """
+
+        if lang == 'nl':
+            return sentiment(self.clean_text())
+        else:
+            return 0.0, 0.0
```

Comment:
I think a not-implemented error should be raised instead of silently returning `0`